frontend/src/pages/journal/components/note/components/note-input/note-input.tsx

```diff
@@ -0,0 +1,86 @@
+import { useCallback, useFormController } from '#libs/hooks/hooks.js';
+import {
+  type FormControl,
+  type FormFieldPath,
+  type FormFieldValues,
+} from '#libs/types/types.js';
+
+import {
+  ContentEditable,
+  LexicalComposer,
+  LexicalErrorBoundary,
+  PlainTextPlugin,
+} from './components/components.js';
+import {
+  $createParagraphNode,
+  $createTextNode,
+  $getRoot,
+} from './libs/helpers/helpers.js';
+import { OnChangePlugin } from './libs/plugins/plugins.js';
+import {
+  type ErrorBoundaryType,
+  type LexicalEditor,
+} from './libs/types/types.js';
+import styles from './styles.module.scss';
+
+type Properties<T extends FormFieldValues> = {
+  name: FormFieldPath<T>;
+  control: FormControl<T, null>;
+  placeholder: string;
+  contentEditableStyle?: string | undefined;
+  onError: (error: Error) => void;
+};
+
+const NoteInput = <T extends FormFieldValues>({
+  placeholder,
+  name,
+  control,
+  contentEditableStyle,
+  onError,
+}: Properties<T>): JSX.Element => {
+  const { field } = useFormController({ name, control });
+  const { value, onChange } = field;
+
+  const handleChange = useCallback(
+    (newValue: string) => {
+      const parser = new DOMParser();
+      const dom = parser.parseFromString(newValue, 'text/html');
+
+      onChange(dom.body.textContent);
```

Comment:
can't we just pass value instead of this?
```suggestion
      onChange(newValue);
```

Reply:
we can, and then the response will be presented in HTML string